websub-ballerina/sub_listener.bal

```diff
@@ -99,19 +100,18 @@ public class Listener {
         self.callbackUrl = retrieveCallbackUrl(
                         configuration?.callback, configuration.appendServiceUrl, 
                         servicePath, self.port, self.listenerConfig);
-        
         if isLoggingGeneratedCallback(configuration?.callback, name) {
             log:printInfo("Autogenerated callback ", URL = self.callbackUrl);
-        }   
-
-        self.httpService = check new(subscriberService, configuration?.secret);
+        }
+        HttpToWebsubAdaptor handler = check new (subscriberService);
```

Comment:
I think we don't need to use the name handler anymore. Shall we rename the variable to `httpToWebsubAdaptor` or at least `adaptor`